ballet/project.py

```diff
@@ -207,6 +207,12 @@ def branch(self):
     def on_master(self):
         return self.branch == 'master'
 
+    def on_master_after_merge(self):
+        if not self.on_master():
+            return False
+        else:
+            return len(self.repo.active_branch.commit.parents) > 1
```

Comment:
I prefer `self.repo.head` but up to you.

Reply:
that pushes the statement to one line so I'll go with this. Thank you!